src/applications/edu-benefits/0994/pages/trainingProgramsInformation.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import fullSchema from 'vets-json-schema/dist/22-0994-schema.json';
+import _ from 'lodash';
+import dateUI from 'us-forms-system/lib/js/definitions/date';
+
+const { plannedStartDate, vetTecProgramLocations } = fullSchema.properties;
+
+const trainingDescription = (
```

Comment:
should this be in a content file?

Reply:
Same here, figured since it was just a few lines, does it need to be in it's own file? I can do it to follow the pattern if you want though. Just let me know.

Reply:
Done